android/src/main/java/com/oney/WebRTCModule/PeerConnectionObserver.java

```diff
@@ -284,10 +319,21 @@ public void onAddStream(MediaStream mediaStream) {
 
     @Override
     public void onRemoveStream(MediaStream mediaStream) {
-        String streamReactTag = webRTCModule.onRemoveStream(mediaStream);
+        String streamId = mediaStream.label();
```

Comment:
Then I have to wrap the line - will change it but honestly this kind of changes are waste of time